### ISSUE TITLE
LinkedIn OG

### ISSUE THREAD

**Issue by dzienisz** [Devographics/Monorepo#157]
when I share
`I got a 86% knowledge score on this year's #StateOfJS survey! Take the survey here: https://survey.devographics.com/survey/state-of-js/2022?source=post_survey_share `
on LinkedIn the picture is of GraphQL survey not JS survey

**Comment by SachaG** [comment]
Yeah, other people have reported that… my feeling is that I have enough work fixing bugs in my own code without worrying about trying to fix LinkedIn :)

**Comment by dzienisz** [comment]
How to change the image

Provide a metadata tag for the og:image in the page's head section. For example:

`<meta name="image" property="og:image" content="[Image URL here]">`

**Comment by SachaG** [comment]
Thanks, I'll look into it for the next round of surveys.

**Comment by dzienisz** [comment]
https://github.com/Devographics/Monorepo/blob/main/homepage/src/helpers/meta.ts OG image
